raiden/ui/cli.py

```diff
@@ -306,4 +306,4 @@ def run(ctx, **kwargs):
         gevent.signal(signal.SIGINT, event.set)
         event.wait()
 
-        app_.stop()
+        app_.stop(graceful=True)
```

Comment:
Shouldn't we have some argument or configuration value setting this? And have it by default on? Also add a test fixture to easily control behaviour in tests.

Also I think you should use said option in all these places (in addition to ui/cli.py:309 which you do here).

1. https://github.com/raiden-network/raiden/blob/5ce06def5c7630f1b33289e8c299dc24905d7ecd/raiden/ui/console.py#L67
2. https://github.com/raiden-network/raiden/blob/5ce06def5c7630f1b33289e8c299dc24905d7ecd/raiden/tests/fixtures/blockchain.py#L186
3. https://github.com/raiden-network/raiden/blob/5ce06def5c7630f1b33289e8c299dc24905d7ecd/raiden/tests/fixtures/raiden_network.py#L20